src/OrchardCore.Modules/OrchardCore.Contents/Indexing/DefaultContentIndexHandler.cs

```diff
@@ -7,20 +7,11 @@ public class DefaultContentIndexHandler : IContentItemIndexHandler
     {
         public Task BuildIndexAsync(BuildIndexContext context)
         {
-            context.DocumentIndex.Set(
-                IndexingConstants.ContentItemIdKey,
-                context.ContentItem.ContentItemId,
-                DocumentIndexOptions.Store);
-
-            context.DocumentIndex.Set(
-                IndexingConstants.ContentItemVersionIdKey,
-                context.ContentItem.ContentItemVersionId,
-                DocumentIndexOptions.Store);
-
+            // Text values are analyzed but stored in field.keyword naturally
             context.DocumentIndex.Set(
                 IndexingConstants.ContentTypeKey,
                 context.ContentItem.ContentType,
-                DocumentIndexOptions.Store);
+                DocumentIndexOptions.Analyze);
```

Comment:
@piedone Here is one breaking change that will require changing Queries in Lucene. This is a necessary change because Elasticsearch automatically maps the ContentType as a tokenized Text and the non-tokenized "keyword" should be used instead.